packages/react-native-gesture-handler/src/RNGestureHandlerModule.web.ts

```diff
@@ -63,7 +63,11 @@ export default {
     }
 
     // @ts-ignore Types should be HTMLElement or React.Component
-    NodeManager.getHandler(handlerTag).init(newView, propsRef);
+    NodeManager.getHandler(handlerTag).init(newView, propsRef, actionType);
+  },
+  detachGestureHandler(handlerTag: number) {
+    shouldPreventDrop = false;
```

Comment:
Have you checked [this PR](https://github.com/software-mansion/react-native-gesture-handler/pull/3125)? Don't we also have to check this flag here?